### PR TITLE
progress bar for freeze time

### PR DESCRIPTION
**Pull request by michaone007**

### CODE REVIEW

**Comment by zakiva** [comment]
@michaone007 need rebase, what's the status here?

**Comment by michaone007** [comment]
After solving conflicts ready to merge

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java [hunk 1]
```diff
@@ -322,4 +330,13 @@ public void run() {
     public void changeBackgroundOpacity (int opacity) {
         activityBackground.getBackground().setAlpha(opacity);
     }
-}
+
+    public void freezeProgressBar() {
+        freezeBar.setVisibility(View.VISIBLE);
```
Comment (zakiva) [inline app/src/main/java/com/example/zakiva/santa/TriviaGame.java:405]: when and where we set it back to GONE?

File: app/src/main/res/drawable/freeze_prog_bar.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<shape
```
Comment (zakiva) [inline app/src/main/res/drawable/freeze_prog_bar.xml]: in terms of colors: please use the existing purple and grey.

File: app/src/main/res/layout/activity_trivia_game.xml [hunk 3]
```diff
@@ -347,4 +343,17 @@
         android:visibility="gone"
         app:srcCompat="@drawable/bonus_screen" />
 
+
+    <ProgressBar
+        android:id="@+id/circular_progress_bar"
+        style="?android:attr/progressBarStyleHorizontal"
+        android:layout_width="40dp"
+        android:layout_height="40dp"
+        android:layout_toLeftOf="@id/timeBox"
```
Comment (zakiva) [inline app/src/main/res/layout/activity_trivia_game.xml]: I'm not sure if this is the right location, can you try to put the circle around the time box?

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java (hunk 1 above)
Comment (michaone007) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java:405]: in xml

File: app/src/main/res/layout/activity_trivia_game.xml (hunk 3 above)
Comment (michaone007) [reply app/src/main/res/layout/activity_trivia_game.xml]: around? what do you mean left right bottom top?

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java (hunk 1 above)
Comment (michaone007) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java:405]: in xml

File: app/src/main/res/drawable/freeze_prog_bar.xml (hunk 2 above)
Comment (michaone007) [reply app/src/main/res/drawable/freeze_prog_bar.xml]: its already purple where exactly do you mean grey, inside the circle?

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java (hunk 1 above)
Comment (zakiva) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java:405]: but after we click freeze and it ends..?

File: app/src/main/res/drawable/freeze_prog_bar.xml (hunk 2 above)
Comment (zakiva) [reply app/src/main/res/drawable/freeze_prog_bar.xml]: no. inside the circle it should be white. the circle itself should be grey, and the progress dynamic line should be purple.

File: app/src/main/res/layout/activity_trivia_game.xml (hunk 3 above)
Comment (zakiva) [reply app/src/main/res/layout/activity_trivia_game.xml]: around means that the time box is inside the circle: center hor & center ver.

**Comment by michaone007** [comment]
ready to merge

File: app/src/main/java/com/example/zakiva/santa/TriviaGame.java (hunk 1 above)
Comment (michaone007) [reply app/src/main/java/com/example/zakiva/santa/TriviaGame.java:405]: done

**Pull request merged**